src/Hit/Issue.hs

```diff
@@ -51,13 +57,16 @@ getIssue num = fetchIssue num >>= \case
     Left err -> errorMessage $ show err
     Right is -> putTextLn $ showIssueFull is
 
-showIssueName :: Text -> Issue -> Text
-showIssueName colorCode Issue{..} =
-    arrow <> colorCode <> " [#" <> show @Text (unIssueNumber issueNumber) <> "] " <> resetCode <> issueTitle
+showIssueName :: Text -> Int -> Issue -> Text
+showIssueName colorCode padSize Issue{..} =
+    arrow <> colorCode <> " [#" <> show (unIssueNumber issueNumber) <> "] " <> padding <> resetCode <> issueTitle
```

Comment:
It would be nice to reuse `padRight` function here. But this probably can make code a bit more difficult :thinking:

Reply:
Yes, I also think padding the entire `[#" <> show (unIssueNumber issueNumber) <> "] "` would result in a code that's more difficult to read and more brittle to changes. So I chose to add trailing spaces manually here.

Anyway, if you think using padRight here is more appropriate, I'm happy to change.